### PR TITLE
Made a conditional delete (if intervals have been modified) prior to merge rules from DB

### PR DESCRIPTION
**Pull request by mrutid**
Made a conditional delete (if intervals have been modified) prior to merge rules from DB

### CODE REVIEW

File: lib/models/noSignal.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ var util = require('util'),
     ID_REGEXP = 5,
```
Comment (fgalan) [inline lib/models/noSignal.js]: Is this PR the fix for issue #253? It should be mentioned in the CNR file (with a reference to issue number)

Comment (mrutid) [inline lib/models/noSignal.js]: ok, let's finish the testing first.

Comment (mrutid) [inline lib/models/noSignal.js]: done.

**Comment by AlvaroVega** [comment]
It works!!

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**